abci/cmd/abci-cli/abci-cli.go

```diff
@@ -600,6 +618,43 @@ func cmdQuery(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func cmdPrepareProposal(cmd *cobra.Command, args []string) error {
```

Comment:
I might also be useful to accept extension... although I guess the complexity wouldn't justify the usefulness

Reply:
@sergio-mena I agree, but I think we would want to rewrite a lot of these tests at some point but right now this is not very high on the priority list. Can you please re-review the code that there is now, is it sufficient to provide basic testing functionality for someone wanting to use the command line for it?